keps/prod-readiness/sig-apps/2232.yaml

```diff
@@ -0,0 +1,3 @@
+kep-number: 2232
+alpha:
+  approver: "@wojtek-t"
```

Comment:
Please add me as an approver.

Reply:
this is PRR

You are already a reviewer https://github.com/kubernetes/enhancements/pull/2234/files#diff-b66dc5abe4613f3e3684dff19de37d48960a8221e08fa10226e116cb1fcfda7aR9

Reply:
yeah, I realized that when i reached end and forgot to remove this, sorry for the noise :sweat_smile:

Reply:
I used to be able to resolve comments - did anything change that I'm no longer able to do that?

Reply:
That's disappointing :disappointed: